packages/ui-stencil/src/components/internal/orama-search-results/orama-search-results.tsx

```diff
@@ -52,7 +67,7 @@ export class SearchResults {
           )}
           <orama-chat-suggestions
             as="list"
-            icon={<Icon name="starFour" size={16} />}
+            icon={<Icon name="starFour" size={16} color="var(--text-color-accent, text-color('accent')" />}
```

Comment:
this can probably be handled in a better way... any suggestions?

Reply:
Well, as teh component is marked as "scoped" instead of "shadow", I think styles on a css query selector by class or tag name would be enough. But it do not see this current approach as something bad, actually